routing/routing_tests/index_graph_tools.hpp

```diff
@@ -25,10 +27,12 @@ namespace routing_test
 {
 using namespace routing;
 
+NumMwmId constexpr kTestNumMwmId = 777;
```

Comment:
Errr, seems that this is less than a total number of valid mwms now, is it expected? If so, why not to set this to zero?